packages/bento-design-system/src/createBentoComponents.ts

```diff
@@ -40,191 +10,100 @@ import {
-  );
-
-  const ContentWithSidebar = createContentWithSidebar(Box);
-
-  const Disclosure = createDisclosure(merge(defaultConfigs.disclosure, config.disclosure ?? {}));
-
-  const DisclosureGroup = createDisclosureGroup(
-    merge(defaultConfigs.disclosureGroup, config.disclosureGroup ?? {}),
-    {
-      Disclosure,
-    }
-  );
-
-  const Feedback = createFeedback(merge(defaultConfigs.feedback, config.feedback ?? {}), {
-    Button,
-  });
-
-  const { Form, FormRow, FormSection } = createFormLayoutComponents(
-    merge(defaultConfigs.formLayout, config.formLayout ?? {}),
-    {
-      Actions,
-    }
-  );
-
-  const Tooltip = createTooltip(merge(defaultConfigs.tooltip, config.tooltip ?? {}));
-
-  const { List, ListItem } = createListComponents(merge(defaultConfigs.list, config.list ?? {}));
-  const Menu = createMenu(merge(defaultConfigs.menu, config.menu ?? {}), { List });
-
-  const {
-    CheckboxField,
-    CheckboxGroupField,
-    Field,
-    NumberField,
-    RadioGroupField,
-    ReadOnlyField,
-    SelectField,
-    SliderField,
-    TextField,
-    TextArea,
-    DateField,
-  } = createFormFields(
-    {
-      dropdown: merge(defaultConfigs.dropdown, config.dropdown ?? {}),
-      field: merge(defaultConfigs.field, config.field ?? {}),
-      input: merge(defaultConfigs.input, config.input ?? {}),
-      selectionControl: merge(defaultConfigs.selectionControl, config.selectionControl ?? {}),
-      slider: merge(defaultConfigs.slider, config.slider ?? {}),
-      date: merge(defaultConfigs.dateField, config.dateField ?? {}),
-    },
-    { Tooltip, IconButton, Menu, Button }
-  );
-
-  const { Modal, CustomModal } = createModal(merge(defaultConfigs.modal, config.modal ?? {}), {
-    Actions,
-    IconButton,
-  });
-
-  const Navigation = createNavigation(merge(defaultConfigs.navigation, config.navigation ?? {}));
-
-  const ProgressBar = createProgressBar(
-    merge(defaultConfigs.progressBar, config.progressBar ?? {})
-  );
-
-  const SearchBar = createSearchBar(merge(defaultConfigs.searchBar, config.searchBar ?? {}), {
-    Field,
-    IconButton,
-  });
-
-  const { Stepper, Step: StepperStep } = createStepper(
-    merge(defaultConfigs.stepper, config.stepper ?? {})
-  );
-
-  const Switch = createSwitch(
-    merge(defaultConfigs.selectionControl.element, config.selectionControl?.element ?? {}),
-    { Field }
-  );
-
-  const Table = createTable(merge(defaultConfigs.table, config.table ?? {}), {
-    Tooltip,
-    Feedback,
-    IconButton,
-  });
-  const tableColumn = createTableColumns({ Button, ButtonLink, IconButton, Chip, Tooltip });
-
-  const Tabs = createTabs(merge(defaultConfigs.folderTabs, config.tabs ?? {}));
-
-  const { Toast, ToastProvider } = createToast(merge(defaultConfigs.toast, config.toast ?? {}), {
-    Button,
-    IconButton,
-  });
-
-  const DesignSystemProvider = createBentoProvider(ToastProvider);
+function internalCreateBentoComponents(
```

Comment:
Here I've tried to preserve the same interface as before. It should be unnoticeable if you just call `createBentoComponents` once.

It will be different if you call it more than once with different configurations, since it then attaches the config to `BentoProvider` and not to the components themselves, so you need to either use `withBentoConfig` or wrap the component in the respective `BentoProvider`.

Not a big deal anyway, it's a simple migration.